### ISSUE TITLE
[Bug] Android 9 and 10 synchronizer not working - reported by nazarealexandru11@gmail.com

### ISSUE THREAD

**Issue by alexandrudsc** [alexandrudsc/University-timetable-Android#25]
This sound pretty bad, I think that long awaited update is required.
```txt
Am descarcat aplicatia, dupa care am vrut sa selectez faculatea din care fac parte( FDSA) si nu mi 
da sa selectez faculatatea. Am instalat si dezinstalat aplicatia de vre o 3 ori, dar fara succes.  Eu 
sunt din anul 2 , si am mai instalat aceasta aplicatie pe un Huawei y 3 si mi a mers, acum pe un 
Samsung A 10 nu merge. Mersi frumos ca ai raspuns.
```

**Comment by alexandrudsc** [comment]
There is a new url orar2.usv.ro. Need to check that

**Comment by alexandrudsc** [comment]
The problem is definitely not with the android component, it is rather related to the endpoint (which seems to not be available anymore)

**Comment by alexandrudsc** [comment]
The endpoint was changed from http://www.orar.usv.ro to http://orar.usv.ro/orar and they are redirected to orar2.usv.ro.   

but #20 should definitely be finished before this

**Comment by alexandrudsc** [comment]
done here 6670402b095fbbffb38025a9dbda0d91cd6d0516  
but #20 and #26 must be done before the new version

**Comment by alexandrudsc** [comment]
will be merged into v1.0.0
